DigitalLearningSolutions.Web/Views/Support/RequestSupportTicket/SuccessPage.cshtml

```diff
@@ -0,0 +1,18 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Support.RequestSupportTicket;
+
+@model FreshDeskResponseViewModel
+@{
+   ViewData["Title"] = "Success";
+}
+<div class="nhsuk-u-margin-bottom-8">
+    <h1 class="nhsuk-heading-xl">Request @Model.TicketId submitted</h1>
+    <p class="nhsuk-body-l nhsuk-u-margin-bottom-6">
+        Your request number is @Model.TicketId. This will appear in any emails relating to the request. You should also quote it in any communication with the support team.
+            </p>
+    <p class="nhsuk-body-l nhsuk-u-margin-bottom-6">
+                You will receive an email when a member of the support team responds to it
+            </p>
+    <a class="nhsuk-button nhsuk-button--secondary" role="button" asp-controller="RequestSupportTicket" asp-action="Index">
```

Comment:
These Back links should use the [NHS Design System back-link component](https://service-manual.nhs.uk/design-system/components/back-link), as per wireframes.

Reply:
This has been implemented and changes are pushed. Please verify.